roseus_smach/src/state-machine.l

```diff
@@ -116,7 +116,9 @@
          (send active-state :execute userdata :step (1- step))))
 |#
    ;; execute once on this machine
-   (let (ret)
+   (let (ret
+         (tmp-active-state nil)
```

Comment:
- You should not name `tmp` or `temp`. I think it's better to use `next-active-state`.
- Also symbol defined by `let` is initialized as `nil` by default. so you can use simply `(let (ret next-active-state)...`